app/src/main/java/org/sagebionetworks/research/mpower/research/DataSourceManager.kt

```diff
@@ -141,17 +152,17 @@ data class StudyBurstConfiguration(
     fun startTimeWindow(now: LocalDateTime): LocalDateTime {
         return now.minusSeconds(expiresLimit)
     }
+}
 
-    /**
-     * @return a random element in the set if any exist
-     */
-    private fun <T> Set<T>?.randomElement(): T? {
-        if (this == null) {
-            return null
-        }
-        if (isEmpty()) {
-            return null
-        }
-        return elementAt(Random().nextInt(size))
+/**
+ * @return a random element in the set if any exist
+ */
+fun <T> Set<T>?.randomElement(): T? {
+    if (this == null) {
+        return null
+    }
+    if (isEmpty()) {
+        return null
     }
+    return elementAt(Random().nextInt(size))
```

Comment:
Does the `Random().nextInt(size)` always return either 0 or 1 where `size==2`?

Reply:
Yes
`The nextInt(int n) is used to get a random number between 0(inclusive) and the number passed in this argument(n), exclusive.`